files/apis/03-06-args.R

```diff
@@ -0,0 +1,22 @@
+#' @post /submit
+#' get /submit
```

Comment:
Can we kill this line? We don't need @get and right now it's missing the `@` to be a real annotation anyways.

Reply:
Done